vunit/ui.py

```diff
@@ -290,14 +290,16 @@ class VUnit(object):  # pylint: disable=too-many-instance-attributes, too-many-p
     """
 
     @classmethod
-    def from_argv(cls, argv=None, compile_builtins=True, vhdl_standard=None):
+    def from_argv(cls, argv=None, compile_builtins=True, vhdl_standard=None, external=None):
```

Comment:
> not add external argument here but instead let the user do compile_builtins=False and then add the builtins explicitly later using add_builtins with the external argument

Done.

> It would have been better not to add the builtins by default 

Do you mean to remove `compile_builtins` from `from_argv` and `from_args`? That'd be a breaking change, wouldn't it?